unime/src/lib/components/molecules/navigation/BottomNavItem.svelte

```diff
@@ -13,16 +13,12 @@
     <div class="h-[4px] w-full rounded-full bg-primary" />
   {/if}
 
-  <!-- Button -->
+  <!-- Active button gets 4px less padding to accommodate indicator. -->
   <button class={`rounded-full px-6 ${active ? 'pt-2' : 'pt-3'}`} on:click={() => dispatch('click')}>
-    <div class="flex h-12 w-10 flex-col items-center justify-center">
-      <!-- Currently, there is no better way of applying conditional Tailwind classes to a named slot inside a component. -->
+    <div class="flex h-12 w-10 flex-col items-center justify-center text-slate-800 dark:text-grey">
+      <!-- Slot accepts SVG icons, which bring their own dimensions. -->
       <slot name="icon" />
-      <div
-        class={`pt-[6px] text-[12px]/[20px] font-medium ${
-          active ? 'text-slate-800 dark:text-grey' : 'text-slate-300 dark:text-slate-500'
-        }`}
-      >
+      <div class="pt-[6px] text-[12px]/[20px] font-medium">
```

Comment:
Should we really drop the styling of the icon based on "active" state? (_see PR comment_)

Reply:
Also fine. I would, however, choose a color with AAA contrast for inactive icons.